grabbit/src/test/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessorSpec.groovy

```diff
@@ -69,4 +75,136 @@ class JcrNodesProcessorSpec extends Specification {
         propertyString.type == STRING
     }
 
+    def "Can marshall a JCR Node with mandatory child node to a Protobuf Message"() {
```

Comment:
I have added test case for a node with mandatory child. It was fun to write the test case as there were few interesting challenges while writing. Could not use Jackalope APIs also i had to write some custom iterators for testing.